fastly/wasm.go

```diff
@@ -0,0 +1,102 @@
+package fastly
+
+import (
+	"fmt"
+	"io"
+	"time"
+)
+
+// WasmPackage is a container for data returned about a Wasm package.
+type WasmPackage struct {
```

Comment:
On second review and thought, I think the public interface for this shouldn't mention Wasm at all. So this file should be renamed `package.go` and this struct `Package{}` and the methods `GetPackage()` etc. Apologies for the late  change, however this would better reflect the api endpoints and the terminology we use in C@E documentation etc.